third-party/glad/KHR/khrplatform.h

```diff
@@ -90,12 +90,20 @@
  *                                  int arg2) KHRONOS_APIATTRIBUTES;
  */
 
+#if defined(__SCITECH_SNAP__) && !defined(KHRONOS_STATIC)
```

Comment:
Why do we need to change a third-party code?

Reply:
Oh did you updat glad version?

Reply:
Because it generates warnings in VS2019 in LibCI, somehow -- I've actually seen them locally, too -- about '#pragma GCC'

Reply:
(note that I didn't change it manually -- I re-generated it)